### DISCUSSION TITLE
Feature Request QR Code as jpg

### DISCUSSION THREAD

**Discussion by DuschdrBabbe** [angristan/wireguard-install#267] [Ideas]
It  would be nice if the script generates also a jpg/png with the Client QR Code 

greetings

Pascal

**Comment by oiuchi** [comment]
@DuschdrBabbe 
Hello ! You can easily add it.
add one string in file "wireguard-install.sh"



`qrencode -t png -o "${HOME_DIR}/${SERVER_WG_NIC}-client-${CLIENT_NAME}.png" -r "${HOME_DIR}/${SERVER_WG_NIC}-client-${CLIENT_NAME}.conf"`


add this after 

```
echo -e "\nHere is your client config file as a QR Code:"

	qrencode -t ansiutf8 -l L <"${HOME_DIR}/${SERVER_WG_NIC}-client-${CLIENT_NAME}.conf"
```


after adding new user ,  png  file with QR wil be in directory with configs.
